cmd/example-posix/main.go

```diff
@@ -150,28 +155,32 @@ func main() {
 			if err != nil {
 				klog.Exitf("Failed to read entry file %q: %q", fp, err)
 			}
-			entries <- entryInfo{name: fp, b: b}
+			entries <- entryInfo{name: fp, e: tessera.NewEntry(b)}
 		}
 		close(entries)
 	}()
 
-	for entry := range entries {
-		// ask storage to sequence
-		dupe := false
-		seq, err := st.Sequence(context.Background(), entry.b)
-		if err != nil {
-			if errors.Is(err, posix.ErrDupeLeaf) {
-				dupe = true
-			} else {
-				klog.Exitf("failed to sequence %q: %q", entry.name, err)
+	numWorkers := 256
+	if l := len(toAdd); l < numWorkers {
+		numWorkers = l
+	}
+
+	wg := sync.WaitGroup{}
+	for i := 0; i < numWorkers; i++ {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			for entry := range entries {
```

Comment:
Can we rename `entries` to `entryChan`? It took me a confused minute to realise this threads were cooperating and not duplicating each others work by iterating over the same _slice_.